torch/distributed/optim/zero_redundancy_optimizer.py

```diff
@@ -158,6 +180,17 @@ class _OverlapInfo():
     This contains the information needed by :class:`ZeroRedundancyOptimizer`
     to overlap with :class:`DistributedDataParallel`.
 
+    Arguments:
+        world_size (int): world size of the process group being used.
+
+    Attributes:
+        uniform_bucket_assignment (bool): if ``True``, then the assignment of
```

Comment:
also mention that the threshold is `overlap_info.total_size / self.world_size`?

Reply:
will renaming this var to `shard_bucket` preserve the same meaning, but shorter?